### ISSUE TITLE
TypeError: load() missing 1 required positional argument: 'Loader'

### ISSUE THREAD

**Issue by UsamaSadiq** [Arello-Mobile/swagger2rst#10]
[pyyaml==6.0](https://github.com/yaml/pyyaml/blob/master/CHANGES#L17) made the `Loader` argument necessary when using `yaml.load()` which is now causing error with the new version of `pyyaml`.
```
Creating file /home/runner/work/django-user-tasks/django-user-tasks/docs/modules.rst.
Traceback (most recent call last):
  File "/home/runner/work/django-user-tasks/django-user-tasks/.tox/docs/bin/swg2rst", line 8, in <module>
    sys.exit(main())
  File "/home/runner/work/django-user-tasks/django-user-tasks/.tox/docs/lib/python3.8/site-packages/swg2rst/swagger2rst.py", line 32, in main
    doc = _parse_file(_file)
  File "/home/runner/work/django-user-tasks/django-user-tasks/.tox/docs/lib/python3.8/site-packages/swg2rst/swagger2rst.py", line 74, in _parse_file
    doc = yaml.load(_file)
TypeError: load() missing 1 required positional argument: 'Loader'
```

**Comment by shaders** [comment]
Thank you.

**Comment by UsamaSadiq** [comment]
Hi @shaders could you please release a new version of the package so we can utilise this fix. Currently, we are blocked on this change. Thanks.

**Comment by shaders** [comment]
released v0.0.5
